packages/dd-trace/test/telemetry/send-data.spec.js

```diff
@@ -12,40 +17,78 @@ describe('sendData', () => {
       '../exporters/common/request': request
     })
   })
+
   it('should call to request (TCP)', () => {
-    sendDataModule.sendData({ hostname: '', port: '12345', tags: { 'runtime-id': '123' } }, 'test', 'test', 'req-type')
+    sendDataModule.sendData({ hostname: '', port: '12345', tags: { 'runtime-id': '123' } },
+      application, 'test', 'req-type')
```

Comment:
very weird line break, this is more js idiomatic:
```suggestion
    sendDataModule.sendData({
      hostname: '',
      port: '12345',
      tags: { 'runtime-id': '123' }
    }, application, 'test', 'req-type')
```

Reply:
@iunanua you didn't update it correctly, look at the suggestion above ^